tests/image/Dockerfile

```diff
@@ -0,0 +1,145 @@
+# docker build -t registry.gitlab.com/altinity-qa/clickhouse/cicd/clickhouse-operator .
+FROM ubuntu:20.04
+
+RUN apt-get update \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install --yes \
+    apt-transport-https \
+    software-properties-common \
+    htop \
+    vim \
+    gettext-base \
+    ethtool \
+    jq \
+    socat \
+    make \
+    gcc \
+    g++ \
+    gpg \
+    ca-certificates \
+    bash \
+    btrfs-progs \
+    e2fsprogs \
+    iptables \
+    xfsprogs \
+    tar \
+    pigz \
+    wget \
+    git \
+    perl \
+    iproute2 \
+    cgroupfs-mount \
+    python3-pip \
+    tzdata \
+    psmisc \
+    libreadline-dev \
+    libicu-dev \
+    bsdutils \
+    curl \
+    liblua5.1-dev \
+    luajit \
+    libssl-dev \
+    libcurl4-openssl-dev \
+    gdb \
+    && rm -rf \
+        /var/lib/apt/lists/* \
+        /var/cache/debconf \
+        /tmp/* \
+    && apt-get clean
+
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+RUN echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
+RUN apt-get update
+RUN apt-get install --yes docker-ce docker-ce-cli containerd.io
+
+RUN curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+RUN echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list
+RUN curl -LO https://storage.googleapis.com/minikube/releases/v1.23.0/minikube-linux-amd64
+RUN install minikube-linux-amd64 /usr/local/bin/minikube
+RUN apt-get update
+RUN apt-get install -y kubectl
+
+ENV TZ=Europe/Moscow
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN pip3 install urllib3 testflows==1.7.4 PyYAML docker-compose==1.29.1 docker==5.0.0 dicttoxml tzlocal python-dateutil setuptools
+
+ENV GOLANG_VERSION 1.16
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F6BC817356A3D45E
+RUN add-apt-repository ppa:longsleep/golang-backports
+RUN apt install --no-install-recommends -y golang-${GOLANG_VERSION}-go
+RUN ln -nvsf /usr/lib/go-${GOLANG_VERSION}/bin/go /bin/go
+RUN ln -nvsf /usr/lib/go-${GOLANG_VERSION}/bin/gofmt /bin/gofmt
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CC86BB64
+RUN add-apt-repository ppa:rmescandon/yq
+RUN apt install --no-install-recommends -y yq
+
+ENV DOCKER_CHANNEL stable
+ENV DOCKER_VERSION 20.10.6
+
+RUN set -eux; \
+  \
+# this "case" statement is generated via "update.sh"
+  \
+  if ! wget -nv -O docker.tgz "https://download.docker.com/linux/static/${DOCKER_CHANNEL}/x86_64/docker-${DOCKER_VERSION}.tgz"; then \
```

Comment:
you already installed docker on line 53, why you setup it again?
if you want modern version of docker just use something like that
```
sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 8D81803C0EBFCD88
echo "deb https://download.docker.com/linux/ubuntu focal test" | sudo tee /etc/apt/sources.list.d/docker.list
apt-get install --no-install-recommends -y docker-ce docker-ce-cli containerd.io
```